File: 1832.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        s = set()

        for letter in sentence:
            s.add(letter)
        
        for i in range(ord('a'), ord('z') + 1):
            if(chr(i) not in s):
                return False
        
        return True

File: 1832.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        unordered_set<char> letters;

        for(int i = 0; i < sentence.size(); i++) {
            letters.insert(sentence[i]);
        }
        for(int i = 97; i <= 122; i++) {
            if(letters.find(i) == letters.end()) {
                return false;
            }
        }
        return true;
    }
};